test/cpp_api_parity/parity-tracker.md

```diff
@@ -110,6 +110,7 @@ torch::nn::PairwiseDistance|Yes|No
 torch::nn::L1Loss|Yes|No
 torch::nn::MSELoss|Yes|No
 torch::nn::CrossEntropyLoss|Yes|No
+torch::nn::CrossEntropyLossWithSoftLabels|Yes|No
```

Comment:
It's been a long time since I've seen the parity tracker. Is there a comment or documentation somewhere about what tests this auto-generates?

Reply:
No docs that I'm aware of :( My understanding is that it does this:

1. Instantiates a module in both C++ / python using the `constructor_args` / `cpp_constructor_args` entries in the `criterion_tests` dict entry corresponding to the module. If those aren't specified, it defaults to no args, which is okay for this loss.
2. It sends `input` / `target` through the C++ / python modules
3. It compares the resulting outputs

I noticed that the parity tracker checks functional forms as well, so I added that.